src/pages/stub.ts

```diff
@@ -0,0 +1,15 @@
+import compileTemplate from './stub.pug';
+import './stub.css';
+import Block from '../utils/block';
+
+export interface ILinkProps {}
+
+export class Stub extends Block {
+    constructor(props: ILinkProps) {
+        super(props);
+    }
+
+    render() {
+        return this.compile(compileTemplate, {...this.props})
```

Comment:
Этот момент тоже стоит привести к одному виду. Важно чтобы код был написан в едином стиле, это улучшает его чтение и сохраняет в порядке
```suggestion
        return this.compile(compileTemplate, { ...this.props })
```